socorro/unittest/external/rabbitmq/test_crashstorage.py

```diff
@@ -263,141 +263,6 @@ def test_ack_crash(self):
             'crash_id'
         )
 
-    def test_new_crash(self):
-        config = self._setup_config()
-        config.transaction_executor_class = \
-            TransactionExecutorWithInfiniteBackoff
-        crash_store = RabbitMQCrashStorage(config)
-
-        iterable = [
-            StopIteration(),
-            ('1', '1', 'crash_id'),
-        ]
-
-        def iter_the_iterable(queue):
-            item = iterable.pop()
-            if isinstance(item, Exception):
-                raise item
-            return item
-
-        crash_store.rabbitmq.return_value.__enter__.return_value  \
-            .channel.basic_get = MagicMock(side_effect=iter_the_iterable)
-
-        expected = 'crash_id'
-        for result in crash_store.new_crashes():
-            assert expected == result
```

Comment:
I think because this test raises `StopIteration()` first, it doesn't actually test anything.

If we fix it so it's not raising `StopIteration()` but instead calling `.next()` on a generator:

```
def iter_the_iterable(queue):
    return iter(iterable).next()
```

then this test fails because it can't acknowledge the crash id because that part isn't mocked out.

I think this is a bad test. +1 to removing it.